nesta/production/orms/meetup_es_config.json

```diff
@@ -29,13 +29,16 @@
                 "_terms_sdg_description": {
                     "type": "keyword"
                 },
+                "booleanFlag_autotranslate_entity": {
```

Comment:
minor, I'd rename to `booleanFlag_autotranslated_entity` or `bool_transl_entity` (the shorter the better [*])

Reply:
Our priority in the naming convention is clear legibility. Our feeling is that `transl` or even `_translated_` has some ambiguity which disappears with adding the `auto` prefix. The `booleanFlag` is preferred over `bool` since this strictly speaking isn't a type, rather an object. There could be non-boolean flags in the future and we'd prefer to have a book-keeping mechanism for this.